src/test/scala/chiselTests/VectorPacketIO.scala

```diff
@@ -51,7 +51,8 @@ class BrokenVectorPacketModule extends Module {
 }
 
 class VectorPacketIOUnitTester extends BasicTester {
-  val device_under_test = Module(new BrokenVectorPacketModule)
+  val dut = Module(new BrokenVectorPacketModule)
+  dut.io <> DontCare
```

Comment:
These tests don't check circuit behavior at all? They just elaborate? (in which case, another PR might be just to have them elaborate instead of run)

Reply:
It looks like it's a test for an old bug in Firrtl, what it really needs is "compileToVerilog".

Anyway this PR was just intended to get my very important Firrtl bugfix to work. It's a pretty scary bug that drops logic on connections to submodule ports. It **must** be in 3.0.1/1.01